### PR TITLE
Add default harmony configs

### PR DESCRIPTION
**Pull request by lawrencexia**

### CODE REVIEW

File: core/chains/evm/config/chain_specific_config.go [hunk 1]
```diff
@@ -267,6 +267,15 @@ func setChainSpecificConfigDefaultSets() {
 	avalancheFuji := avalancheMainnet
 	avalancheFuji.linkContractAddress = "0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846"
 
+	// Harmony
+	harmonyMainnet := fallbackDefaultSet
+	harmonyMainnet.linkContractAddress = "0x218532a12a389a4a92fC0C5Fb22901D1c19198aA"
+	harmonyMainnet.gasPriceDefault = *big.NewInt(5000000000)
```
Comment (samsondav) [inline core/chains/evm/config/chain_specific_config.go:273]: Is this Gwei? You can use `assets.GWei(5)` to make it clearer.

File: core/chains/evm/config/chain_specific_config.go [hunk 2]
```diff
@@ -267,6 +267,15 @@ func setChainSpecificConfigDefaultSets() {
...
+	harmonyMainnet.minIncomingConfirmations = 3
```
Comment (samsondav) [inline core/chains/evm/config/chain_specific_config.go:274]: What are the finality rules on harmony? Is it probabilistic?

**Review by samsondav — CHANGES_REQUESTED** [review]

File: core/chains/evm/config/chain_specific_config.go (hunk 2 above)
Comment (aalu1418) [reply core/chains/evm/config/chain_specific_config.go:274]: Good question! 

No it's not probabilistic, it requires 2/3 majority quorum on a block for finality. According to them it's finality in 2 seconds. 
https://docs.harmony.one/home/general/technology/transactions

Do you think we could lower the `minIncomingConfirmations` ?

File: core/chains/evm/config/chain_specific_config.go (hunk 1 above)
Comment (lawrencexia) [reply core/chains/evm/config/chain_specific_config.go:273]: good call, thank you!

File: core/chains/evm/config/chain_specific_config.go (hunk 2 above)
Comment (samsondav) [reply core/chains/evm/config/chain_specific_config.go:274]: It's not about time it's about how many blocks. How many blocks until finality on Harmony?

Comment (aalu1418) [reply core/chains/evm/config/chain_specific_config.go:274]: according to them 1 block finality
image

Comment (samsondav) [reply core/chains/evm/config/chain_specific_config.go:274]: Then, this can safely be set to 1

Comment (lawrencexia) [reply core/chains/evm/config/chain_specific_config.go]: Will do! Can y'all take a look again on Monday to see if there's any additional changes you recommend?

**Review by samsondav — APPROVED** [review]
LGTM, test failures on this are likely to be flakes (something we need to improve).

Pls try running them again.

**Pull request merged**